boards/pic32-clicker/Makefile.features

```diff
@@ -0,0 +1,9 @@
+# Put defined MCU peripherals here (in alphabetical order)
+FEATURES_PROVIDED += periph_timer
+FEATURES_PROVIDED += periph_uart
+
+# Various other features (if any)
+FEATURES_PROVIDED += cpp
+
+# The board MPU family (used for grouping by the CI system)
+FEATURES_MCU_GROUP = mips-pic32mx
```

Comment:
You should combine the MCU groups into one, since we only have support for three MIPS boards, including this PR.

Change all of the MIPS boards to the same group, e.g. `FEATURES_MCU_GROUP=mips` or `FEATURES_MCU_GROUP=mips32r2`.

This grouping is used when building in the CI system, as well as when compile testing locally:
`BUILDTEST_MCU_GROUP=mips make buildtest` will do a compilation test for all MIPS boards only.